source/modulo_controllers/src/RobotControllerInterface.cpp

```diff
@@ -196,6 +196,20 @@ rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn RobotC
   return CallbackReturn::SUCCESS;
 }
 
+rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn RobotControllerInterface::on_deactivate() {
+  if (!control_type_.empty()) {
+    if (control_type_ == hardware_interface::HW_IF_POSITION) {
+      for (std::size_t index = 0; index < joints_.size(); ++index) {
+        previous_joint_command_values_[index] = joint_state_.get_position(index);
+      }
```

Comment:
is this necessary, since it is already applied from last write loop? I suppose it's nice because it locks in the current position, not the last command.

Reply:
At the same time you are making a good point because the position control case is already handled in line 290 so an outdated position command will neither be decayed nor applied again unless we actually no have a `new_joint_command_ready_`